src/main/java/uk/gov/justice/hmpps/casenotes/config/WebClientConfiguration.kt

```diff
@@ -23,34 +24,43 @@ import org.springframework.web.reactive.function.client.WebClient.Builder
 import reactor.netty.http.client.HttpClient
 import reactor.netty.tcp.TcpClient
 import uk.gov.justice.hmpps.casenotes.utils.UserContext
+import java.time.Duration
 
 @Configuration
-class WebClientConfiguration(private val prop: OffenderCaseNoteProperties) {
+class WebClientConfiguration(
+    /** Elite2 API Base URL endpoint ("http://localhost:8080") */
+    @Value("\${elite2.api.base.url}") private val elite2ApiBaseUrl: @URL String,
+    /**  OAUTH2 API Rest URL endpoint ("http://localhost:9090/auth/api") */
+    @Value("\${oauth.api.base.url}") private val oauthApiBaseUrl: @URL String,
+    /** OAUTH2 API Rest URL endpoint ("http://localhost:8100") */
+    @Value("\${tokenverification.api.base.url}") private val tokenVerificationApiBaseUrl: @URL String,
+    @Value("\${api.health-timeout:1s}") private val healthTimeout: Duration) {
+
   @Bean
-  fun elite2ApiWebClient(builder: Builder): WebClient = createForwardAuthWebClient(builder, prop.elite2ApiBaseUrl)
+  fun elite2ApiWebClient(builder: Builder): WebClient = createForwardAuthWebClient(builder, elite2ApiBaseUrl)
 
   @Bean
-  fun elite2ApiHealthWebClient(builder: Builder): WebClient = createHealthClient(builder, prop.elite2ApiBaseUrl)
+  fun elite2ApiHealthWebClient(builder: Builder): WebClient = createHealthClient(builder, elite2ApiBaseUrl)
 
   @Bean
-  fun oauthApiWebClient(builder: Builder): WebClient = createForwardAuthWebClient(builder, prop.oauthApiBaseUrl)
+  fun oauthApiWebClient(builder: Builder): WebClient = createForwardAuthWebClient(builder, oauthApiBaseUrl)
 
   @Bean
-  fun oauthApiHealthWebClient(builder: Builder): WebClient = createHealthClient(builder, prop.oauthApiBaseUrl)
+  fun oauthApiHealthWebClient(builder: Builder): WebClient = createHealthClient(builder, oauthApiBaseUrl)
 
   @Bean
-  fun tokenVerificationApiWebClient(builder: Builder): WebClient = createForwardAuthWebClient(builder, prop.tokenVerificationApiBaseUrl)
+  fun tokenVerificationApiWebClient(builder: Builder): WebClient = builder.baseUrl(tokenVerificationApiBaseUrl).build()
```

Comment:
don't use forward header here as calls to the web client happen before the filter is configured